static/js/components/ProgramSelector.js

```diff
@@ -51,24 +49,24 @@ export default class ProgramSelector extends React.Component {
     const {
       currentProgramEnrollment,
       programs: { programEnrollments },
-      dashboard: { programs },
     } = this.props;
 
     let currentId;
     if (!_.isNil(currentProgramEnrollment)) {
       currentId = currentProgramEnrollment.id;
     }
 
-    const sortedProgramEnrollments = _.sortBy(programEnrollments, 'title');
+    const sortedProgramEnrollments = _.sortBy(programEnrollments, 'title').filter(
+      enrollment => enrollment.enrolled
+    );
```

Comment:
i think it would make a lot more sense to first sort the program enrollments, then create the lists for `sortedProgramEnrollments` and `unenrolledPrograms` (instead of doing the same operations on the same list in different ways). eg:

```javascript
const sortedPrograms = _.sortBy(programEnrollments, 'title');
let enrolledPrograms = programEnrollments.filter(program => program.enrolled);
let unenrolledPrograms = programEnrollments.filter(program => !program.enrolled);
```